pkg/k8s/utils/workload.go

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Istio Authors
+// Copyright Authors of Hubble
+// Copyright Authors of Cilium
+
+// GetWorkloadMetaFromPod and cronJobNameRegexp are copied from
+// https://github.com/istio/istio/blob/1aca7a67afd7b3e1d24fafb2fbfbeaf1e41534c0/pkg/kube/util.go
+//
+// Modifications:
+// GetDeployMetaFromPod has been renamed to to GetWorkloadMetaFromPod and has
```

Comment:
```suggestion
// GetDeployMetaFromPod has been renamed to GetWorkloadMetaFromPod and has
```

Also, typo in commit log (`istio/isto/...`)

Reply:
Ah good catch. Fixed the typos.